README.md

```diff
@@ -1,104 +1,130 @@
-# nucypher-ts
+ A TypeScript library for interacting with access control functionality in the browser.
 
-Communicate with NuCypher network from browser.
+ Full Documentation can be found [here](https://docs.threshold.network/app-development/threshold-access-control-tac).
 
-## Disclaimer
+> :warning: 
+> `nucypher-ts` is under [active development](https://github.com/nucypher/nucypher-ts/pulls).
+>
+> SDK does not support policy revocation.
+>
+> We expect breaking changes.
+# Get Started with TAC
 
-**This is a work in progress**
+The following interactive tutorial is a quick way for developers to familiarize themselves with the Threshold Access Control service. Before jumping into the following, we recommend reading this introduction to [Conditions Based decryption](https://docs.threshold.network/fundamentals/threshold-access-control/conditions-based-decryption-cbd) and its key concepts. CBD is the primary technology underpinning Threshold Access Control and directly enables data sharing predicated on the fulfillment of predefined conditions.
 
-- SDK does not support policy revocation.
-- We expect breaking changes.
+## 1. Install nucypher-ts
 
-## Supported networks
+We'll kick things off by installing `nucypher-ts` – a TypeScript library for interacting with access control functionality in the browser. The APIs for leveraging most TAC functionality are contained in nucypher-ts.
 
-`nucypher-ts` is available on the following networks:
+```
+yarn add @nucypher/nucypher-ts
+```
+
+## 2. Build a Cohort
 
-- Polygon
-- Mumbai (Polygon testnet)
+Next, we'll parametrize a _Cohort_ to correspond to our risk preferences. Cohort objects delineate the group of independent nodes that will collectively provide access control service to a given data sharing flow. _Threshold_ and _Shares_ are two parameters used to construct a Cohort. For example, a `3-of-5` Cohort requires responses – the delivery of shares – from a minimum of 3 out of a total of 5 Cohort members in order to reconstruct the original plaintext data.
 
-## Usage
+We generate a Cohort object by:
 
-Run with:
+```javascript
+import { Cohort } from '@nucypher/nucypher-ts';
 
-```bash
-yarn install
-yarn test
-yarn build
+const config = {
+  threshold: 3,
+  shares: 5,
+  porterUri: 'https://porter-lynx.nucypher.community',
+};
+const newCohort = await Cohort.create(config);
 ```
 
-## Development
+Notice that we have also provided a `porterUri`. **Porter is a web-based service that interacts with nodes on the network on behalf of applications – an "Infura for TAC".** In this example, we've chosen a `lynx` (testnet) Porter endpoint.
 
-Install git hooks
+## 3. Create Conditions
 
-```bash
-npx husky install
-```
+We will now specify the conditions on which data access will be predicated – i.e. what will the data requester need to prove in order to gain decryption rights. In this tutorial, nodes will check that the requester owns a specific ERC721 NFT:
```

Comment:
Made a similar comment in Manu's gitbook change request - should we point out that ERC721Ownership object is wrapper that ultimately utilizes the ERC721 `ownerOf` function? It would provide more context to what the condition is doing, and explain the reason for the `parameters`

Reply:
It's covered in more detail on the conditions page https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions#conditions.erc721ownership

Reply:
I've added a note which also directs people to the Conditions reference page

Reply:
Manu has made some changes (not yet merged) to the wording in this regard as well - https://app.gitbook.com/o/R2meumXNNad4y1B10iL7/s/WosjlL4zUGUMlcMfuSAp/~/changes/9qUH3dwSnLSjWbz13dlr/app-development/threshold-access-control-tac/get-started-with-tac